### PR TITLE
#1017 : Adding visible announcements button for easy visibility

### PR DESCRIPTION
**Pull request by abhiramr**
### Issue Reference
This PR addresses the Issue : Fixes part of #1017 
https://github.com/IEEEKeralaSection/rescuekerala/issues/1017

### Summary
dded new Red button for Announcements on top for easy access and visibility per #1017 .

### CODE REVIEW

**Comment by abhiramr** [comment]
@tomahawk-pilot I've tested it locally, if that helps.

**Comment by subins2000** [comment]
I've tested this locally. Here's how it looks :
Screenshot_2019-08-10_01-10-18

**Comment by tomahawk-pilot** [comment]
@subins2000 is verified

**Comment by tomahawk-pilot** [comment]
@biswaz please verify and merge this

**Comment by abhiramr** [comment]
This satisfies #1017 with hashtags that can be cleared and added while adding a new announcement - 

hashtags1
hashtags2
hashtags3

**Comment by abhiramr** [comment]
Here's how it looks when a tag has been selected - 

Screenshot from 2019-08-10 03-29-50

**Comment by jojurajan** [comment]
@abhiramr Could you take a look at #1027 and merge the changes to your PR if applicable?

**Comment by abhiramr** [comment]
@jojurajan I've looked at them already. My PR covers all of it and a little more and unfortunately renders #1027 superfluous. I really appreciate his work though :)

**Comment by abhiramr** [comment]
@tomahawk-pilot This still hasn't been merged no?

**Comment by subins2000** [comment]
@abhiramr I'm getting this error tho :
Screenshot_2019-08-10_12-27-49

**Comment by abhiramr** [comment]
@subins2000 Surprising. Do you have my complete code? Have migrations been rerun?

**Comment by subins2000** [comment]
@abhiramr Never mind. It works. It didn't work cause no tags were added.

Note to sysadmin : Make sure atleast one announcement item has a tag after merging this, otherwise the whole page will fail

Screenshot_2019-08-10_12-40-10

**Comment by abhiramr** [comment]
@subins2000  Thanks for this test. Will try to add a case for 0 tags. Ideally shouldn't fail because of no data.

**Comment by tomahawk-pilot** [comment]
is the model modified to add the tags
@subins2000

**Comment by abhiramr** [comment]
@tomahawk-pilot  Yes it is.

**Comment by subins2000** [comment]
@tomahawk-pilot yes, the model has been modified by @abhiramr

**Comment by subins2000** [comment]
There seems to be a conflict now

File: mainapp/views.py [hunk 1]
```diff
@@ -744,16 +744,39 @@ def find_people(request):
 
     return render(request, 'mainapp/people.html', {'filter': filter , "data" : people })
 
+#Get unique hashtags from items in DB
+def get_hashtags(announcement_obj):
+    hashtags_str = ""
+    for i in (announcement_obj.objects.all().values_list('hashtags', flat=True)):
```
Comment (biswaz) [inline mainapp/views.py]: why are you converting a list into a string and then back into a list again ?

File: mainapp/views.py [hunk 2]
```diff
@@ -744,16 +744,39 @@ def find_people(request):
...
+        if i !='':
+            hashtags_str = hashtags_str +","+i
+    hashtags = list(set([j.strip() for j in hashtags_str.strip(',').split(',')]))
```
Comment (biswaz) [inline mainapp/views.py]: similarly list into a set and then into a list again

**Comment by abhiramr** [comment]
I'll give you reasons in a bit. For now can you edit any single announcement with a tag? I see a 500 on the page. @biswaz

**Comment by biswaz** [comment]
@abhiramr Yeah I was in the process of doing it. It's done now. We could've actually used the taggit lib. But if it works.. it works :)

**Comment by sreenadh** [comment]
How about using a plugin like bootstrap-tagsinput along with typeahead for the tag input field.

https://bootstrap-tagsinput.github.io/bootstrap-tagsinput/examples/ see the example with typeahead. Make sure that it doesn't compromise the UX on small screens

**Pull request merged**